src/index.ts

```diff
@@ -2,25 +2,39 @@ import path from "path";
 import * as vortexApi from "vortex-api"; // eslint-disable-line import/no-extraneous-dependencies
 
 // Our stuff
-import { EPICAPP_ID, GAME_ID, GOGAPP_ID, STEAMAPP_ID } from "./index.metadata";
+import {
+  EPICAPP_ID,
+  GAME_ID,
+  GOGAPP_ID,
+  STEAMAPP_ID,
+} from "./index.metadata";
 import { wrapTestSupported, wrapInstall, internalPipelineInstaller } from "./installers";
 import { VortexExtensionContext, VortexGameStoreEntry } from "./vortex-wrapper";
+import { wrappedPrepareForModding } from './extension';
 
 const moddingTools = [
   {
-    id: "CyberCat",
-    name: "CyberCAT Save Editor",
-    shortName: "CyberCAT",
-    logo: "SaveEditor.png",
-    executable: () => path.join("CyberCAT", "CP2077SaveEditor.exe"),
+    id: `CyberCat`,
+    name: `CyberCAT Save Editor`,
+    shortName: `CyberCAT`,
+    logo: `SaveEditor.png`,
+    executable: () => path.join(`CyberCAT`, `CP2077SaveEditor.exe`),
     requiredFiles: [
-      path.join("CyberCAT", "CP2077SaveEditor.exe"),
-      path.join("CyberCAT", "licenses", "CyberCAT.Core.LICENSE.txt"),
+      path.join(`CyberCAT`, `CP2077SaveEditor.exe`),
+      path.join(`CyberCAT`, `licenses`, `CyberCAT.Core.LICENSE.txt`),
     ],
     defaultPrimary: true,
     shell: false,
     relative: true,
   },
+  {
+    id: `REDLauncher`,
+    name: `REDLauncher`,
```

Comment:
Should these be called _pre_ launcher also? Was that the term used in the docs?

Reply:
I'm not sure???

Reply:
CDPR call it REDlauncher when we spoke to them.